cli/run/commandPlugins.ts

```diff
@@ -63,10 +63,10 @@ function loadAndRegisterPlugin(inputOptions: InputOptions, pluginText: string) {
 			}
 		}
 	}
-	if (typeof plugin === 'object' && pluginText in plugin) {
-		// some plugins do not use `export default` for their entry point.
-		// attempt to use the plugin name as the named import name.
-		plugin = plugin[pluginText];
+	// some plugins do not use `module.exports` for their entry point,
+	// in which case we try the named default export and the plugin name
+	if (typeof plugin === 'object') {
+		plugin = plugin.default || plugin[pluginText]
```

Comment:
I took the liberty to simplify the logic to get rid of an untested case and thus make coverage pass. After all, I think it does not really matter if we fail because we are calling an object or because we are calling `undefined` if neither case applies.